spikeextractors/extractors/cellexplorersortingextractor/cellexplorersortingextractor.py

```diff
@@ -0,0 +1,102 @@
+from spikeextractors import SortingExtractor
+import numpy as np
+from pathlib import Path
+from spikeextractors.extraction_tools import check_valid_unit_id
+from typing import Union
+from scipy.io import loadmat, savemat
+
+PathType = Union[str, Path, None]
```

Comment:
```suggestion
from typing import Optional
from scipy.io import loadmat, savemat

PathType = Optional[str, Path]
```

Reply:
Actually, I think it's just better if we remove the None inclusion here so it's clearer that it's non-optional. This was copied over from a structure that supported multiple path options, but we don't have that here.